src/services/config.service.ts

```diff
@@ -85,30 +114,30 @@ const loadConfig = async (configLocation: string) => {
   });
 
   if (!response.ok) {
-    throw new Error('Failed to load the config');
+    throw new Error('Failed to load the config. Please check the config path and the file availability');
   }
 
   const data = await response.json();
 
-  enrichConfig(data);
+  if (!data) {
+    throw new Error('No config found');
+  }
 
-  return data;
+  return enrichConfig(data);
 };
 
 /**
- * Add content default options
+ * Add default values to the config
  * @param {Config} data
  */
-const enrichConfig = (data: Config) => {
-  if (!data.content.some(({ type }) => type === PersonalShelf.Favorites)) {
-    data.content.push({ type: PersonalShelf.Favorites });
-  }
+const enrichConfig = (config: Config): Config => {
+  const { content, siteName } = config;
+  const updatedContent = content.map((content) => Object.assign({ enableText: true, featured: false }, content));
 
-  data.content = data.content.map((content) => Object.assign({ enableText: true, featured: false }, content));
-  data.siteName = data.siteName || 'My OTT Application';
+  return { ...config, siteName: siteName || 'My OTT Application', content: updatedContent };
```

Comment:
Might be nice to put this into the translations and also expose the build version

Reply:
I added the default name to translations. What do you mean by "build version"? Is it the one we use for tagging our versions on GitHub now?

Reply:
Yes, but that can be a separate feature request